src/pages/settings/Payments/PaymentsPage/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import BasePaymentsPage from './BasePaymentsPage';
+
+const PaymentsPage = () => (
+    <BasePaymentsPage listenResize />
```

Comment:
Not going to leave a full review here, but please see this draft PR which explains proper usage of boolean props to enable or disable features...

https://github.com/Expensify/App/pull/7611

@rushatgabhane please 👀  and thank you 🙇

Reply:
And applied this would be changed to something like `shouldListenForResize`.

Reply:
Agreed. Let's change it.

Reply:
Thanks, I agree!